src/admin-views/settings/tickets-commerce/gateways/container.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Template to display a list of featured gateways.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this              Template object.
+ * @var Gateway[]                    $gateways          Array of gateway objects.
```

Comment:
Gateway is not a class like that you need to insert a `use .../Gateway;` after the docblock.